### DISCUSSION TITLE
Any advice to UX designers that would like to contribute?

### DISCUSSION THREAD

**Discussion by scottjenson** [jupyter/notebook#6663] [General]
I went to your contributor guidelines and didn't see any mention of UX. Are you interested in UX designers helping out? If so, what would you suggest?

**Comment by jtpio** [comment]
Thanks @scottjenson for your interest :+1: 

> Are you interested in UX designers helping out?

Yes!

Indeed the contributing docs don't mention anything about UX. The [JupyterLab documentation](https://jupyterlab.readthedocs.io/en/latest/developer/contributing.html) also doesn't seem to have guidelines for UX contributions, although there are definitely some people working on UX for this project.

Ideally the contributing guide should be updated to include guidelines. In the meantime contributions are welcome, thanks!

**Comment by jtpio** [comment]
Linking to a relevant issue: https://github.com/jupyterlab/team-compass/issues/168
